standard/info/wikis/osu/info.lua

```diff
@@ -0,0 +1,45 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 2007,
+	wikiName = 'osu',
+	name = 'osu!',
+	games = {
+		osu = {
+			abbreviation = 'osu!',
+			name = 'osu!',
+			link = 'osu!',
+			logo = {
+				darkMode = 'osu! allmode.png',
+				lightMode = 'osu! allmode.png',
```

Comment:
Not sure if can/should commit to the branch so only leaving a comment: can we instead use this [Osu! stable allmode.png](https://liquipedia.net/commons/File:Osu!_stable_allmode.png) as the logo for the stable version? Since the current [osu! allmode.png](https://liquipedia.net/commons/File:Osu!_allmode.png) is only used for the lazer game client.

Reply:
cc @Hesketh2